assets/js/views/edit.js

```diff
@@ -0,0 +1,29 @@
+var EditView = Backbone.View.extend({
+  template: AppTemplates.edit,
+
+  events: {
+    'remove entry': 'deletePost'
```

Comment:
Remember when registering events it is the event type you want to listen for `submit`, `click`, `mouseOver`, etc and then what selector you want to listen to. For instance if you wanted to listen for clicks on a button with a class of `superman` to call a `fly` function in your view, then your events hash would look like:

```
events: {
  'click button.superman': 'fly'
}
```

So you should be able to update your events to make the delete post work.

The other thing you need to do here is to listen for a form to submit (just like in your create view) but instead of creating a new record in your collection, you need to update the properties of your current `this.model` and then save it.